content-services/7.3/support/index.md

```diff
@@ -6,6 +6,116 @@ Here is a list of the individual components that have been through the complete
 
 Choose a combination of products to build your own Supported Stack. If anything is unclear then please [contact Support]({% link support/latest/contact.md %}).
 
+{% capture seven-three-two %}
+
+| Version | Notes |

```

Comment:
There are several missing expected supported platform upgrades here (e.g.: IE/SS, ATS, Salesforce Connector etc.)

If the goal of this PR is to just add the Keycloak compatibility then that's fine, but in that case a separate task to upgrade the 7.3.2 supported platform with the expected components should be worked on eventually.

Reply:
This PR is only to add Keycloak, the rest will be addded separately. Thank you for your review 👍